src/token-controls/styles/token-controls.scss

```
@@ -0,0 +1,284 @@
+	display: inline-flex;
+	flex: 0 0 auto;
+	align-items: center;
+	justify-content: center;
+	width: 16px;
+	height: 16px;
+	color: #949494;
+
+	svg {
+		width: 16px;
+		height: 16px;
+		fill: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)